public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'formik';
+import SubHeader from '../../../../../components/SubHeader';
+import { FormikSelect } from '../../../../../components/FormControls';
+
+
+const methodOptions = [{ value: 'POST', text: 'POST' }, { value: 'PUT', text: 'PUT' }, { value: 'PATCH', text: 'PATCH' }];
+
+
+
+const propTypes = {
```

Comment:
Is the type checking effective here?
I thought the usage is either `MethodEditor.propTypes = { ... };`, or adding an additional line `MethodEditor.propTypes = propTypes;` besides the `const propTypes = { ...}`

Reply:
Thanks! Went with `MethodEditor.propTypes = propTypes;` to be consistent with the other editors.